### PR TITLE
feat: Add support to build python wheels

### PR DESCRIPTION
**Pull request by flub**
The added CI job builds the wheels for linux, windows and osx.  These wheels
have been uploaded to pypi.

### CODE REVIEW

File: src/.github/workflows/wheels.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Build Python Wheels
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+  jobs:
```
Comment (Arqu) [inline src/.github/workflows/wheels.yml:9]: This should be indented one level less

Comment (flub) [reply src/.github/workflows/wheels.yml:9]: :scream: yaml is the worst

Comment (Arqu) [reply src/.github/workflows/wheels.yml]: given you're building python only makes sense indentation plays a role :P screw `{}`, right?

**Review by b5 — APPROVED** [review]
LGTM!

**Pull request merged**